package.json

```diff
@@ -32,12 +32,16 @@
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-qunit": "^1.4.0",
     "ember-cli-release": "0.2.8",
+    "ember-cli-sass": "5.3.1",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-data": "^2.4.2",
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
-    "loader.js": "^4.0.1"
+    "loader.js": "^4.0.1",
+    "ember-cli-coffeescript": "^1.13.2",
+    "ember-disable-proxy-controllers": "^1.0.1",
```

Comment:
Do we need this? It appears to be a node package to assist in preparing for upgrading Ember to 2.0 which we're well past now.